bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import random
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours, ERROR_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog that creates personal bookmarks by relaying a message to the user's DMs."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, jump_url: str = None, *args) -> None:
+        """Bookmarks a message."""
+        if jump_url is not None and jump_url != "*":
+            if "discordapp.com/channels" not in jump_url:
+                embed_error_1 = discord.Embed(
+                    title=random.choice(ERROR_REPLIES),
+                    description="I can't find the associated message. "
+                                "This command supports the following syntax:\n"
+                                "`[command alias] [message url] (bookmark name)`",
+                    colour=Colours.soft_red
+                )
+                await ctx.send(embed=embed_error_1)
+                return
+        if jump_url is None or jump_url == "*":
+            channel = ctx.channel
+            async for message in channel.history(limit=2):
+                jump_url = message.jump_url
+
+        embed = discord.Embed(
+            title="Your bookmark",
+            description=None,
+            colour=Colours.soft_green
+        )
+        hint = ' '.join(args)
+
+        if hint == "":

```

Comment:
You can use the fact that strings are implicitly False if they are empty
```suggestion
        if not hint:
```